jiratools/assignee.py

```diff
@@ -0,0 +1,18 @@
+"""Issue Reassignment command."""
+import argparse
+
+import jira
+
+from .utils import get_client
+from .helpers import check_for_valid_user, get_issue_by_id
+
+
+def cli_reassign(args: argparse.Namespace) -> None:
+    """Change issue assignment."""
+    client = get_client()
+    get_issue_by_id(args.jira_id)
+    check_for_valid_user(args.user)
+    try:
+        client.assign_issue(args.jira_id, args.user)
+    except jira.exceptions.JIRAError as e:
+        print('ERROR: "{}" trying to assign a new user to the JIRA.'.format(e.text))
```

Comment:
Could be FF, but it'd be nice to include the user and JIRA id in the error message.
From a onesy-twosy on the command line, yeah, you'll "know" which user and JIRA, but if you are scripting this (even from the command line) and iterating over a bunch of JIRAs reassigning them (say because, cough, a coworker left), you won't know which one the failure refers to.